commands/birthday.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+from collections import defaultdict
+import datetime
+import json
+import re
+
+from commands.base import Command
+from helpers import CommandFailure
+
+BIRTHDAY_FILE = "files/birthdays.json"
+BIRTHDAY_ROLE_NAME = "Birthday!"
+
```

Comment:
PEP8 requires 2 newlines after imports/constants if you wish to follow it.

Reply:
Done